src/main/duke/commands/CFind.java

```diff
@@ -0,0 +1,32 @@
+package main.duke.commands;
+
+import main.duke.DukeException;
+import main.duke.TaskList;
+import main.duke.Ui;
+import main.duke.enums.CommandType;
+import main.duke.tasks.Task;
+
+import java.util.ArrayList;
+
+public class CFind extends Command{
+    protected String findString;
+
+    public CFind(String findString) {
+        super(CommandType.FIND);
+        this.findString = findString;
+    }
+
+    public String getFindString() { return this.findString; }
```

Comment:
I suggest follow EgyptianStyle when we make a function which has one line only. I noticed the same issue in several other places too.